app/controllers/admin/base_controller.rb

```diff
@@ -1,5 +1,16 @@
 # frozen_string_literal: true
 
 class Admin::BaseController < ApplicationController
+  include Pundit
+
   before_action :authenticate_user!
+  before_action :ensure_admin
+  after_action :verify_authorized, except: :index
```

Comment:
Rubocop is very unhappy about this, saying
```
app/controllers/admin/base_controller.rb:8:3: C: Rails/LexicallyScopedActionFilter: index is not explicitly defined on the class. (https://rails.rubystyle.guide#lexically-scoped-action-filter)
					after_action :verify_authorized, except: :index
```

Do I just define an `index` here? Seems messy